cmd/export-client/res/configuration.toml

```diff
@@ -13,6 +13,6 @@ ConsulHost = 'localhost'
 ConsulPort = 8500
 CheckInterval = '10s'
 ConsulProfilesActive = 'go'
-ClientHost = 'EXPORT_CLIENT_HOST'
-DistroHost = 'EXPORT_CLIENT_DISTRO_HOST'
+ClientHost = 'edgex-export-client'
```

Comment:
Change to localhost, as the default configuration file assumes that all services are running in the same machine.